weed/pb/master.proto

```diff
@@ -51,6 +51,8 @@ service Seaweed {
   }
   rpc RaftRemoveServer (RaftRemoveServerRequest) returns (RaftRemoveServerResponse) {
   }
+  rpc VolumeGrow (AssignRequest) returns (VolumeGrowResponse) {
```

Comment:
create a VolumeGrowRequest and remove unnecessary fields

Reply:
done